pkg/cmd/server/start/start_master.go

```diff
@@ -563,21 +563,11 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 			glog.Fatalf("Could not get client for HPA controller: %v", err)
 		}
 
-		_, _, recyclerClient, err := oc.GetServiceAccountClients(bootstrappolicy.InfraPersistentVolumeRecyclerControllerServiceAccountName)
```

Comment:
the recycler SA has a custom role and gets added to the `hostmount-anyuid` SCC, which is required for recycling if I understand correctly. is the binder controller doing all of those things now? if so, it needs the appropriate role and scc